responsibleai/model_monitor/components/src/io_utils.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""This file contains utilities to read write data."""
+
+import mltable
+import pandas as pd
+
+
+def load_mltable_to_df(mltable_path: str) -> pd.DataFrame:
```

Comment:
Do we need this function for a line of code. Why don't you call mltable.load(mltable_path).to_pandas_dataframe() directly from the run.py?

Reply:
honestly this is copied from the azureml-asset repo; they have a special function for writing from mltable to df. IMO it doesn't make too much of a difference either way...one way its a little more readable and the other it's one less function call. Leaving for now unless you're strongly opposed.

Reply:
yeah, it is weird in the sense that you have maintain an extra function with docs extra. Not strongly opposed to this.